docs/Azure_IoT_Integration_Guide/Configure_integration_templates/Azure_integration_templates.md

```diff
@@ -36,42 +36,50 @@ Follow the instruction below to learn how to create integration templates and th
     !!! Example
         `Manufacturer` is unlikely to change over time, so it is a **Property**. `Reboot` is an executable **Command**. `Battery Level` is more ambiguous: it can be a property if we just want to have the actual value. But because we want to have its historical values, it is a **Telemetry**.
 
-Click **Next step**.
+5. Click **Next step**.
 
-Check the Summary and click **Save template**.
+6. Check the Summary and click **Save template**.
 
 ## Assign a template to a group
 
-Now that you have a template, you need to assign it to a group. Learn more about how to create groups in Coiote DM.
+Now that you have a template, you need to assign it to a group. Here is how you create groups in Coiote DM:
 
 1. Go to **Administration —> Azure integration** and see the list of Azure integration templates that exist in your domain.
 2. To assign a template to a specific group, click on the context menu next to the template name and select **Assign to group**.
 
 ![Assign template to group](images/templ_assign.png "Assign template to group"){: .center }
 
-A pop-up window will appear asking you to specify the group that needs to be assigned to this template. Select and click Confirm.
+A dialog box will appear asking you to specify the group that needs to be assigned to this template. Select and click Confirm.
 
-![Assign template to group- pop-up window](images/templ-assign-group.png "Assign template to group- pop-up window"){: .center }
+![Assign template to group- dialog](images/templ-assign-group.png "Assign template to group- dialog"){: .center }
 
 !!! note
     One template can be assigned to more than one group.
 
 Now the devices in the **group ‘anjay’** will report data to Azure according to the way defined in the **template ‘abc_test’**.
 
+### Integration templates in action
+
+As you now know, the Azure integration template is based on a data model which consists of the objects and resources available on your reference device. Coiote DM sends data to Azure every time a device data model is changed, provided that you have set an Observation on the objects and resources in this data model. (Read about setting an Observation in [Azure IoT Hub](/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Set_an_Observation/) and [Azure IoT Central](/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Set_an_Observation/).)
+
+Let’s illustrate the process with two examples. In the first case, the change in the data model is triggered by a device. When the value of the `Battery level` resource changes, the device sends **Send** or **Notify** messages via LwM2M. The device information in Coiote DM is then updated and translated to Device Twin JSON, which in turn is sent to Azure.
```

Comment:
This is true, but only when Battery level is a property in Template. When it is marked as Telemetry, Coiote uses device 2 cloud message. 

I think we should explain that here.